modules/core/client/controllers/core.client.controller.header.spec.js

```diff
@@ -10,7 +10,7 @@
 
     beforeEach(module('core'));
 
-    beforeEach(inject(function(_$state_, _$rootScope_, $controller, _menuService_,_$mdSidenav_) {
+    beforeEach(inject(function(_$state_, _$rootScope_, $controller,_$mdSidenav_, _lodash_) {
```

Comment:
Space after controller,

I don't think you need lodash for anything in the test